apps/src/blockly/addons/blockSvgFrame.js

```diff
@@ -0,0 +1,149 @@
+import msg from '@cdo/locale';
+import color from '@cdo/apps/util/color';
+import {frameSizes} from './cdoConstants.js';
+export default class BlockSvgFrame {
+  constructor(block, text, className, textColor, headerColor, baseColor) {
+    this.block_ = block;
+    this.text = text || msg.block();
+    this.className = className || 'blocklyFrame';
+    this.textColor = textColor || color.white;
+    this.headerColor = headerColor || color.light_gray;
+    this.baseColor = baseColor || color.lightest_gray;
```

Comment:
It's not. I found I was thinking of these as default values but wanted to provide the option to override them when we create a frame (or subclass this). I noticed that we've used different colors for unused/function SVGs because the blocks are semi-transparent for unused. My thinking was that it would be easier to list this here and make it easy to customize later than to hardcode it down below as "lighter_gray" without really showing the relationship between this and other colors.  Open to ideas here!

Reply:
You could still assign it here (ie, `this.baseColor = color.lightest_gray`) without having the extra constructor parameter (and add that ability to customize via constructor later if we end up needing it). I'd vote to not have unused parameters in function definitions generally, but up to you